package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "@rolemodel/turbo-confirm",
   "type": "module",
-  "version": "2.0.2",
+  "version": "2.1.0",
   "description": "Drop-in upgrade for Rails' data-turbo-confirm feature to support custom HTML dialogs.",
   "main": "src/index.js",
+  "dependencies": {
+    "@hotwired/turbo": ">= 8.0.6"
```

Comment:
Should this be a `dependency` or a `peerDependency`/`devDependency`?

Reply:
@Malachirwin that's a good question!  Honestly, I don't know.  Do you have an opinion?

Reply:
@Malachirwin What are the implications of a peerDependency vs. a dependency.  I'm pretty confident that I would not call it a devDependency..

Reply:
What about you @BenEinwechter / @seanpk ?

Reply:
Based on:
> Rather than maintain support for multiple Turbo versions, I believe we can rely on package.json's built-in dependency management. (Please advise if you believe this assumption is incorrect)

You probably would want a peer dependency since what you have would allow someone to specify Turbo 7.x and TurboConfirm 2.1. Javascript works different than ruby. It will resolve this package twice and give you two instances of turbo. doing it as a peer dependency will give a warning when you install saying that it is unmet.

Reply:
Typically peer dependencies would be used when referring to frameworks that you use in a package

Reply:
Thanks @Malachirwin & @justwiebe !!  updated

Reply:
opinions on minor vs patch version bump for this change? @Malachirwin / @justwiebe

Reply:
Agreed on using `peerDependency` (which it looks like is the case).

I would probably consider the least restrictive that is necessary (e.g. `>= 8.0.0`) unless there's a real need in specifying the patch version. That said, peer dependencies are also a lot looser than other dependencies, since you can have "unmet" — so I think it's safe enough to specify the patch version here. This seems fine to me. 👍

Reply:
8.0.6 is the patch version that required this change.

Your comment about peer dependencies being allowed has me rethinking this, though.  This code will not work at all, with Turbo 8.0.4 or below.  The integration point with Turbo changed.  So if someone can upgrade TurboConfirm to this new version without Turbo also being upgraded to at least 8.0.6 then I don't think `peerDependency` is the way to go.

Reply:
I would still argue for a peer dependency. As a dependency, there's a chance that downstream consumers would be running two versions of Turbo side by side. It's pretty common for libraries to specify the minimum version they work with as a peer dependency, it's just then the job of the developers to make sure they're actually looking at the warnings

https://nodejs.org/en/blog/npm/peer-dependencies#the-solution-peer-dependencies
> Some way of saying, "I only work when plugged in to version 1.2.x of my host package, so if you install me, be sure that it's alongside a compatible host." We call this relationship a peer dependency.

Reply:
That's super annoying that Turbo isn't following Semver though